src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -94,7 +94,7 @@ public override bool Equals(DataViewType other)
             return false;
         }
 
-        public override string ToString() => "Text";
+        public override string ToString() => "String";
```

Comment:
>String [](start = 46, length = 6)

This one is a little tricky, since of course the type as we see a few lines above is *not* `String` at all, but rather `ReadOnlyMemory<char>`.

Reply:
Well, it would be internally `ReadOnlyMemory`, but for user who construct class would be fine with `String` for person who works with `TextLoader` we have `DataKind.String`

---
In reply to: [268027832](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268027832) [](ancestors = 268027832)

Reply:
That's fine, just so long as you've thought about it. Though, incidentally, this is not an "internal" detail. We provide convenience conversions to and from string for the sake of our reflection based API, but `IDataView` itself works exclusively over `ReadOnlyMemory<char>`, never `string`.

---
In reply to: [268261843](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268261843) [](ancestors = 268261843,268027832)